frontend/app/tsconfig.json

```diff
@@ -20,6 +20,6 @@
     "jsx": "react",
     "resolveJsonModule": true
   },
-  "include": ["**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx"],
+  "include": ["**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx", "src/layouts/PreviewLayout"],
```

Comment:
it's a little odd to add specific folders and files in here, can we look for a more global solution?